### PR TITLE
check for min tls version

### PR DESCRIPTION
**Pull request by nineinchnick**
Check for min tls version, similar to how we check for max classification.

Remove `usesLatestTLSversion` since it's not precise enough what's "latest".

### CODE REVIEW

File: pytm/pytm.py [hunk 1]
```diff
@@ -830,6 +848,12 @@ class Element:
         required=False,
         doc="Maximum data classification this element can handle.",
     )
+    minTLSVersion = varTLSVersion(
+        TLSVersion.NONE,
+        required=False,
+        doc="""Minimum required TLS version required.
+Note that currently only TLS 1.2 and 1.3 are considered secure.""",
```
Comment (izar) [inline pytm/pytm.py]: just wondering if the comment about the versions is not better suited for the actual threat. First, the versions will probably move and we will have to chase them here, second, on a more meta level, this adds a level of judgement that i don't think is present in other attributes (off memory, I didn't check)

Comment (izar) [reply pytm/pytm.py]: Yep, I think this needs changing.

**Review by izar — COMMENTED** [review]
Just need to fix that message.

File: pytm/pytm.py (hunk 1 above)
Comment (nineinchnick) [reply pytm/pytm.py]: I removed the note and I noticed that `required` is used twice.

**Pull request merged**